File: Algorithm/Programmers/Level2/solution(1).py
# 최댓값과 최솟값

def solution(s):
    answer = ''
    s_num = s.split(' ')
    num = []
    
    for i in s_num:
        num.append(int(i))
    
    num = sorted(num)
    
    answer = str(num[0]) + ' ' + str(num[-1])
    
    return answer

File: Algorithm/Programmers/Level2/solution(1).java
// 최댓값과 최솟값

import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        String[] strarr = s.split(" ");
        int n = strarr.length;
        int[] intarr = new int[n];
        
        for (int i = 0; i < n; i++){
            intarr[i] = Integer.valueOf(strarr[i]);
        }
        
        Arrays.sort(intarr);
        
        answer = String.valueOf(intarr[0]) + ' ' + String.valueOf(intarr[n-1]);
        
        return answer;
    }
}